components/src/lib/components/ModalFooter.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import { ButtonPosition, ModalButtonLayout } from "./modalType";
+import { ReactNode } from "react";
+
+export interface FooterProps {
+  buttonLayout?: ModalButtonLayout;
+  buttonPosition?: ButtonPosition;
+  children: ReactNode;
+}
+
+const ModalFooter = ({
+  buttonLayout = "row",
+  buttonPosition = "right",
+  children,
+}: FooterProps) => {
+  return (
+    <ButtonContainer
+      $buttonLayout={buttonLayout}
+      $buttonPosition={buttonPosition}
+    >
+      {children}
+    </ButtonContainer>
+  );
+};
+
+const getButtonPosition = {
```

Comment:
변하지 않는 상수 객체이니, uppercase를 쓰면 어땠을까 싶기도 하네요!
이부분은 각자마다 선호도가 있는 것 같아서요 😸 

```suggestion
const BUTTON_POSITION_TABLE ={
```